4-projects/modules/folder_environments/README.md

```diff
@@ -10,6 +10,7 @@
 
 | Name | Description |
 |------|-------------|
+| dev\_folder\_id |  |
```

Comment:
The folder environments module should probably be removed now, as in this update to the code base the environment folder is created at the top of the hierarchy in the 2-environment steps